tools/build_h3_tools.sh

```diff
@@ -169,3 +169,38 @@ autoreconf -i
   LDFLAGS="${LDFLAGS}"
 ${MAKE} -j $(nproc)
 sudo ${MAKE} install
+
+# boringssl
+echo "Building boringssl..."
+
+# We need this go version.
+GO_BASE_PATH=/usr/local
```

Comment:
Meh, I think /usr/local for go is fine. That's where our CI docker images install it, so I vote for keeping it consistent with them.

Reply:
For docker /usr/local is fine, but this script might be run on any host as well.  If it's in /usr/local it could preempt an existing golang install and leave the user in a state of astonishment.  imo, if this script is going to always install this version of go it should be out of the way of any possible system installations.